src/Config/DatabasePrimitives/DatabaseObject.cs

```diff
@@ -130,6 +130,7 @@ public class ParameterDefinition
 {
     public Type SystemType { get; set; } = null!;
     public DbType? DbType { get; set; }
+    public SqlDbType? SqlDbType { get; set; }
```

Comment:
Can you add a comment that covers the discussion about this only being valid for MSSQL and null for other DBs, with future refactoring potentially to come?

Reply:
https://github.com/Azure/data-api-builder/issues/2128
https://github.com/Azure/data-api-builder/issues/2127

contain references to the discussion for your question.